SUSYBSMAnalysis/HSCP/test/AnalysisCode/Analysis_CommonFunction.h

```diff
@@ -508,14 +513,14 @@ TH3F* loadDeDxTemplate(string path, bool splitByModuleType){
       Prob_ChargePath->RebinX(Prob_ChargePath->GetNbinsX());
    }
 
-   for(int i=0;i<=Prob_ChargePath->GetXaxis()->GetNbins()+1;i++){
+   for(int i=1;i<=Prob_ChargePath->GetXaxis()->GetNbins();i++){// do not include under/overflow here
```

Comment:
why the hell do you decide to change this behaviour?
Under and Over flow bins should be considered